README.md

```diff
@@ -48,70 +48,33 @@ Following diagram illustrate the architecture of the Custos Software.
 * Maven 3.6.x
 
 #### Clone the repository
-  ```
-    git clone -b develop https://github.com/apache/airavata-custos.git
-    
-  ```
-
-#### Build source code
-  
-  Following  command builds the Custos source code and create two docker images of custos_core_server and custos_integration_server
-  
-  ```
-    cd airavata-custos
-    mvn clean install
-  ```
-  
-#### Run Custos on docker
-  
-Following command starts Custos main services and its depend services. All services are listed below and you should be able to
-access them locally if all services are correctly started.
-
-  - Dependent Services
-    * Keycloak (http://localhost:8080/auth/)
-    * MySQL (0.0.0.0:3306)
-    * HashiCorp Vault (http://localhost:8201/)
-    * CILogon (Not available for local development)
-    
-  - Custos Services
-    * Custos Core Service (0.0.0.0:7001 (grpc port))
-    * Custos Integration Service (0.0.0.0:7000 (grpc port))
-    * Custos Rest Proxy (http://localhost:10000(envoy proxy)
-    
-  ```
-     cd custos-utilities/ide-integration/src/main/containers
-     docker-compose up
-  ```
-
-#### Bootstrapping Custos  Super Tenant
-  
-If all services were successfully ran. Custos bootstrap service needs to be run to create a  Super tenant to launch Custos Portal
-   ```
-    cd custos-utilities/custos-bootstrap-service/
-    mvn spring-boot:run
-   ```
-The above command should create the super tenant and it outputs super tenant credentials. Copy those credentials to configure
-Custos Portal.
-
-```
-Note: Make sure to clean up old databases for fresh start.
+```sh
+git clone https://github.com/apache/airavata-custos.git
 ```
 
-#### Install Custos Portal Locally
+#### Start Docker Containers
+Navigate to `/compose`, and start the following containers:
+- Keycloack (http://localhost:8080)
+- Custos DB (MySQL, http://localhost:3306)
+- Vault (http://localhost:8200)
+- Adminer (http://localhost:18080)
 
-Follow the following link to access portal deployment instructions
+```sh
+docker compose up
+```
 
-[Custos Portal](https://github.com/apache/airavata-custos-portal/blob/master/README.md)
+#### Configure Vault
+1. Go to the Vault's exposed port (http://localhost:8200) and walk through the configuration process. 
+   2. You'll need to save your initial root token and unsealed key.
+3. Place your root token in `/application/src/main/resources/application.yml`, on line 50.
 
-You have to configure following properties in the .env file
+1. Install all dependencies through maven.
```

Comment:
include the commands to do these operations (install and running the spring boot application)